internal/parser/scanner/rule_based_scanner_test.go

```diff
@@ -169,26 +169,26 @@ func TestRuleBasedScanner(t *testing.T) {
 		},
 		{
 			"numeric literals",
-			"7 7.5 8.9.8 8.0 0.4 10 10000 18907.890 1890976.09.977",
+			"7 7.5 8.9 .8 8.0 0.4 10 10000 18907.890 1890976.09 .977",
```

Comment:
why did you change these tests?

Reply:
Similar ones were existing already.

Reply:
Reverted. Doesnt make much of a difference, those points are covered.